server/pulp/server/agent/direct/services.py

```diff
@@ -34,7 +34,7 @@ def init():
 
     @staticmethod
     def start():
-        url = config.get('messaging', 'url')
+        url = Services.get_url()
```

Comment:
Using get_url() is a temporary work around until pulp 3.0 when we can change the pulp configuration files.